libraries/tools/kotlin-gradle-plugin-integration-tests/src/test/kotlin/org/jetbrains/kotlin/gradle/AppleFrameworkIT.kt

```diff
@@ -78,6 +78,27 @@ class AppleFrameworkIT : BaseGradleIT() {
         }
     }
 
+    @Test
+    fun `check that macOS framework has symlinks`() {
+        with(Project("sharedAppleFramework")) {
+            val options: BuildOptions = defaultBuildOptions().copy(
+                customEnvironmentVariables = mapOf(
+                    "CONFIGURATION" to "debug",
+                    "SDK_NAME" to "macosx",
+                    "ARCHS" to "x86_64",
+                    "TARGET_BUILD_DIR" to "no use",
+                    "FRAMEWORKS_FOLDER_PATH" to "no use"
+                )
+            )
+            build("assembleDebugAppleFrameworkForXcodeMacosX64", options = options) {
+                assertSuccessful()
+                assertTasksExecuted(":shared:assembleDebugAppleFrameworkForXcodeMacosX64")
+                assertFileExists("/shared/build/xcode-frameworks/debug/macosx/sdk.framework/Headers")
+                assertFileIsSymlink("/shared/build/xcode-frameworks/debug/macosx/sdk.framework/Headers")
```

Comment:
This test checks success build but we need to check macos framework copying.
You can do it this way:
```
customEnvironmentVariables = mapOf(
  "CONFIGURATION" to "debug",
  "SDK_NAME" to "macosx",
  "ARCHS" to "x86_64",
  "TARGET_BUILD_DIR" to workingDir.absolutePath,
  "FRAMEWORKS_FOLDER_PATH" to "build/xcode-derived" //<-- path to embedding assembled framework
)
build(":shared:embedAndSignAppleFrameworkForXcode", options = options) {
  assertSuccessful()
  assertTasksExecuted(":shared:assembleDebugAppleFrameworkForXcodeMacosX64")
  assertFileExists("/build/xcode-derived/sdk.framework/Headers")
  assertFileIsSymlink("/build/xcode-derived/sdk.framework/Headers")
}
```

Reply:
Alright will check this later today. The current test verifies the first copy step to the `xcode-frameworks` folder which was already missing the symlinks.

Reply:
Updated so it's now verifying that the symlinks exists in both the `build/xcode-frameworks` folder and the Xcode derived data folder.